### PR TITLE
use new release process for 1.2 release

### PR DESCRIPTION
**Pull request by njr-11**
Switch to new release process for version 1.2

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/01-compatible-certification-request.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+name: Compatible Certification Request
+about: Start a request for a compatible certification 
+title: 'MicroProfile Rest Client [Version] Compatible Certification Request'
```
Comment (Emily-Jiang) [inline .github/ISSUE_TEMPLATE/01-compatible-certification-request.md:4]: should be MicroProfile Context Propagation

Comment (njr-11) [reply .github/ISSUE_TEMPLATE/01-compatible-certification-request.md]: Thanks for spotting this. I corrected it in the update that also updates the asciidoctor version to match what mp rest client uses.

File: api/pom.xml [hunk 2]
```diff
@@ -27,4 +27,132 @@
...
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-baseline-maven-plugin</artifactId>
+                <version>4.2.0</version>
+                <configuration>
+                    <base>
+                        <version>1.4</version>
```
Comment (Emily-Jiang) [inline api/pom.xml:141]: This should be 1.0

Comment (njr-11) [reply api/pom.xml]: Thanks - it is fixed in commit https://github.com/eclipse/microprofile-context-propagation/pull/207/commits/515eaaa1bed6e3e9c49725750668b45a9ee4e1ea

**Review by Emily-Jiang — APPROVED** [review]

**Pull request merged**